index.html

```diff
@@ -140,11 +115,15 @@ <h2 data-key="repo-id"><a href="https://www.github.com/heremaps/{repo-id}" targe
 			$.getJSON('https://api.github.com/orgs/heremaps/repos')
 			.done(function (repo_data) {
 				repo_data.forEach(function (data, index) {
+					/* The GitHub API will return more repos than the ones we expose here.
+						 Target only existing repo sections.*/
 					var $target = $('#repo-info-' + data.name);
 					if ($target.length > 0) {
-						$target.find('[data-role="repo-lastUpdate"]').text(moment(data.updated_at).format('ll'));
-						$target.find('[data-role="fork-count"]').text(data.forks_count);
-						$target.find('[data-role="star-count"]').text(data.stargazers_count);
+						$target.find('[data-key="repo-type"]').text(data.language);
```

Comment:
Hmm, not sure we mean the same thing here. IMO we should use whatever language we hard-code in JS, unless it is `null`, in which case we use whatever the GitHub API returns. So basically hard-coding `null` means "auto-detect". To start with, we should have `null` everywhere, and only where the GitHub API also returns `null` we hard-code whatever language we want in JS. Does that make sense?